### ISSUE TITLE
Error when using display_heatmap 

### ISSUE THREAD

**Issue by Litschi123** [philipperemy/keract#103]
Hi,

I am getting this error when using the display_heatmap method. 
```
keract.py in display_heatmaps(activations, input_image, directory, save, fix)
    350                 img = img.resize((input_image.shape[1], input_image.shape[0]), Image.LANCZOS)
    351                 img = np.array(img)
--> 352                 axes.flat[i].imshow(input_image / 255.0)
    353                 # overlay the activation at 70% transparency  onto the image with a heatmap colour scheme
    354                 # Lowest activations are dark, highest are dark red, mid are yellow

AttributeError: 'AxesSubplot' object has no attribute 'flat'
```
The shapes of *activations* and *input_image* are bot (1, 28, 28, 1).

**Problem description**
When i was looking at the code i noticed that the the for loop (line 321) which is iterating over the filters is trying to access the individual 'AxesSubplot' objects via array indexing. The problem i see is that in my case only one 'AxesSubplot' object is present and there is no array to be accessed by indexing. 

**Problem solving**
During investigation i tried to solve the error and came up with the following solution:
```
for layer_name, acts in activations.items():
    nrows = int(math.sqrt(acts.shape[-1]) - 0.001) + 1  # best square fit for the given number
    ncols = int(math.ceil(acts.shape[-1] / nrows))
    
    fig1, axes = plt.subplots(nrows, ncols, figsize=(6, 6));
    fig1.suptitle(layer_name)
            
    for i in range(nrows * ncols):
        if not isinstance(axes, numpy.ndarray):
            axes.imshow(acts[0,:,:,i])
            axes.axis('off')
        else:
            if i < acts.shape[-1]:
                axes.flat[i].imshow(acts[0,:,:,i])
                axes.flat[i].axis('off')
            axes.flat[i].axis('off')
```
Obviously this is only part of a final solution. Pleas let me know if i made something wrong completely or if i am on the right track to solve an issue here and make a pull request. Btw this would be my first ever pull request.

Thank You

**Comment by philipperemy** [comment]
Duplicate to https://github.com/philipperemy/keract/issues/105

**Comment by philipperemy** [comment]
Fixed in https://github.com/philipperemy/keract/commit/48ebbc7ac13e1a104eafe3ce7fd799d361664a1e

**Comment by philipperemy** [comment]
Released in 4.2.2
